### DISCUSSION TITLE
想知道未来会支持tags在首页显示的功能吗

### DISCUSSION THREAD

**Discussion by dhe090** [LIlGG/halo-theme-sakura#471] [Ideas]
![image](https://github.com/LIlGG/halo-theme-sakura/assets/47225023/d8dff43e-e525-4c09-893e-660fb0ef4875)
类似这样的可以在首页选择tag的界面

**Comment by LIlGG** [comment]
Sakura 不会支持这样的界面，目前 Sakura 主题基本功能和页面已经定型了。这种界面与现有风格不是很匹配。

**Reply by dhe090** [reply]
好的谢谢，顺便建议下在使用教程里把如何调出tag界面和归档界面的方式写一下，
小白找了半天才发现要自己添加/archives和/tags才能调出这两个页面

**Reply by dhe090** [reply]
然后不知道是我这边的问题还是什么，使用文档的链接是打不开的

**Comment by LIlGG** [comment]
> 好的谢谢，顺便建议下在使用教程里把如何调出tag界面和归档界面的方式写一下， 小白找了半天才发现要自己添加/archives和/tags才能调出这两个页面
 
这两个页面是由 Halo 提供的。不过似乎文档里面确实没有体现，你可以前往 https://github.com/halo-dev/docs/issues 提一下。
